src/hooks/user.ts

```diff
@@ -10,8 +10,8 @@ export default function useUser() {
     await userStore.logout();
     const currentRoute = router.currentRoute.value;
     Message.success('登出成功');
-    router.push({
-      name: logoutTo && typeof logoutTo === 'string' ? logoutTo : 'login',
```

Comment:
Here it seems that it cannot be reduced to True

Reply:
ts should not need to judge the type.